code/datums/components/fantasy/suffixes.dm

```diff
@@ -100,6 +100,8 @@
 		// Some types to remove them and their subtypes
 		possible_mobtypes -= typecacheof(list(
 			/mob/living/carbon/human/species,
```

Comment:
shouldnt it be /mob/living/carbon? carbon xenos dont have ai, thats why /mob/living/simple_animal/hostile/alien exists